### DISCUSSION TITLE
Is there any way to use this software read storage data(SSD or HDD) instead of ram data?

### DISCUSSION THREAD

**Discussion by kucingkembar** [xhluca/dl-translate#50] [Q&A] [answered]
hi, it me again, and sorry about my bad English,
I have a project to use this software for windows tablets with 4GB of ram,
the problem is the ram consumption using this software is quite high, about 2,3GB,
is there any way to use this software read storage data(SSD or HDD) instead of ram data?

thank you for reading, and have a nice day

**Comment by xhluca** [comment, marked as answer]
Hi unfortunately it's not possible. This package uses large transformer models, which require a lot of RAM to store the model as well as the input matrices. I would recommend taking a look at the Marian project if you are planning to mainly use CPU and have RAM restriction: https://marian-nmt.github.io/

**Comment by xhluca** [comment]
I'm moving this to Discussions Q&A.
